src/extension/app-schema/sample-data-access-test/src/test/java/org/geoserver/test/SampleDataAccessWfsTest.java

```diff
@@ -175,8 +174,7 @@ public void prettyPrint(Document doc, OutputStream out) throws Exception {
 
     @Override
     protected void setUpTestData(SystemTestData testData) throws Exception {
-        // TODO Auto-generated method stub
-        
+        testData.setUpSecurity();
```

Comment:
Tests that do not need test data better add security config, otherwise the time saved by not configuring data will be lost to the security migrations. I've added this line to all tests I could locate that do not need pre-canned data